string.c

```diff
@@ -11341,17 +11354,29 @@ enc_str_scrub(rb_encoding *enc, VALUE str, VALUE repl, int cr)
 
 /*
  *  call-seq:
- *    str.scrub -> new_str
- *    str.scrub(repl) -> new_str
- *    str.scrub{|bytes|} -> new_str
+ *    scrub(replacement_string = default_replacement) -> string or self
+ *    scrub{|bytes| ... } -> string or self
+ *
+ *  Returns a copy of self with each invalid byte sequence replaced
+ *  by a replacement string; returns +self+ if nothing there were
+ *  no invalid byte sequences.
+ *
+ *  With no block given and no argument, replaces each invalid sequence
+ *  with the default replacement string
+ *  (<tt>"\uFFFD"</tt> ("�") for a Unicode encoding, <tt>'?'</tt> otherwise):
+ *
+ *    "foo\x81\x81bar".scrub # => "foo��bar"
+ *
+ *  With no block given and argument +replacement_string+ given,
+ *  replaces each invalid sequence with that string:
+ *
+ *    "foo\x81\x81bar".scrub('xyzzy') # => "fooxyzzyxyzzybar"
+ *
+ *  With a block given, replaces each invalid sequence with the value
+ *  of the block:
  *
- *  If the string is invalid byte sequence then replace invalid bytes with given replacement
- *  character, else returns self.
- *  If block is given, replace invalid bytes with returned value of the block.
+ *    "foo\x81\x81bar".scrub {|bytes| 'XYZZY' } # => "fooXYZZYXYZZYbar"
```

Comment:
Might be a good idea to show the use of `bytes` in the example, so readers can better understand what is yielded to the block.

Reply:
Done.